File: Problems/PizzaParty_8/PizzaParty.py
while True:
  people = input("How Many People? ")
  pizzas = input("How Many Pizzas do you have? ")
  if people.isnumeric() and pizzas.isnumeric(): 
    break
  else:
    print("Please enter two positive numbers")
    
peopleInt = int(people)
pizzasInt = int(pizzas)

pieces = (pizzasInt * 8) / peopleInt;
leftovers = (pizzasInt * 8) % peopleInt;

print("%i people with %i pizzas" % (peopleInt, pizzasInt))
print("Each person gets %i pieces of pizza.\n" % (pieces))
print("There are %i leftover pieces.\n" % (leftovers))

File: Problems/PizzaParty_8/PizzaParty.cpp
#include <iostream>
#include <string>
#include <stdlib.h>

using namespace std;

int main()
{
	string people, pizzas;
	int peopleInt, pizzasInt, leftover, pieces;
	
	cout << "How Many People? ";
	cin >> people;
	cout << "How Many Pizzas do you have? ";
	cin >> pizzas;
	
	peopleInt = atoi(people.c_str());
	pizzasInt = atoi(pizzas.c_str());
	
	pieces = (pizzasInt * 8) / peopleInt;
	leftover = (pizzasInt * 8) % peopleInt;
	
	cout << peopleInt << " people with " << pizzasInt << " pizzas\n";
	cout << "Each person gets " << pieces << " pieces of pizza.\n";
	cout << "There are " << leftover << " leftover pieces.\n";
} 

/*TO DO: Check if input is valid*/
